apps/home/components/followed_artists/templates/index.jade

```diff
@@ -2,13 +2,14 @@
 
```

Comment:
We're 'borrowing' the homepage component that lets you follow/search for artists. There's just a couple overrides we need to add (hide header/context, use a passed-in list vs featured artists). Stopped short of doing a bigger component refactor.